pkg/node/wbt/operations/wbt_ops.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operations
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/dell/csi-baremetal/pkg/base/command"
+)
+
+const (
+	wbtValuePath = "/sys/block/%s/queue/wbt_lat_usec"
+	// isWBTEnabledCmdImpl is a CMD to check current WBT value
+	checkWbtValueCmd = "cat " + wbtValuePath
```

Comment:
Do we realy need to use system tools like `cat` and `echo` to read from or write to file? How about go library?

Reply:
created TODO - https://github.com/dell/csi-baremetal/issues/653